src/index.html

```diff
@@ -2,12 +2,56 @@
 <html lang="en">
   <head>
     <meta charset="UTF-8">
-    <meta name="viewport"
-          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <meta name="viewport" content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <title>Product cards</title>
-    <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" >
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="./styles/index.scss">
   </head>
   <body>
-    <h1>Product cards</h1>
+    <div
+    class="card"
+    data-qa="card"
+    >
+      <img
+        src="../src/images/imac.jpeg"
+        alt="imac"
+        class="card__photo"
+      >
+
+      <p class="card__description">
+        APPLE A1419 iMac 27" Retina <br>5K Monoblock (MNED2UA/A)
+      </p>
+
+      <p class="card__product-code">
+          Product code: 195434
+      </p>
+
+      <div class="card__rating">
+        <ul class="stars">
+          <li class="stars__star"></li>
+          <li class="stars__star"></li>
+          <li class="stars__star"></li>
+          <li class="stars__star"></li>
+          <li class="stars__star"></li>
+        </ul>
+        <p class="card__review">Review: 5</p>
+      </div>
+
+
+      <div class="card__price">
+        <p class="card__price card__price--text">Price:</p>
+        <p class="card__price card__price--value">$2,199</p>
+      </div>
+
+      <a
+        href="http://"
```

Comment:
think this should be `href="#"`, it is good practice to point hash for empty urls